doc/swish/http.tex

```diff
@@ -1106,6 +1106,9 @@ \subsection{WebSocket Protocol}\label{sec:websocket-protocol}
 objects.  The \var{wr} procedure is equivalent to
 \code{(lambda (\var{op} \var{x} \var{indent}) (json:write \var{op} \var{x} \var{indent} \var{custom-write}))}.
 
+The default value of \var{custom-write} is the value stored in the
```

Comment:
The description of sorting the keys by string<? needs to be updated to talk about the new process parameter.

Check the description of json:write-object, too.

Reply:
Good find. I tried to update the relevant places.